google-analytics-setup.html

```diff
@@ -0,0 +1,91 @@
+<link rel="import" href="../polymer/polymer.html">
+<link rel="import" href="../google-signin/google-signin-aware.html">
+<link rel="import" href="../google-apis/google-client-loader.html">
```

Comment:
Dashboard is back. It has a design flaw which will probably not happen in production:
- if dashboard updates `google-analytics-chart` properties before `google-analytics-chart` is .ready(), `google-analytics-chart` properties will not propagate to google-analytics-query. Still investigating why.
- my workaround is not to set any chart properties before user is authenticated. User authentication should happen before `analytics-chart` upgrade, but it is not guaranteed.

Now I have two demos, one with dashboard, one without. Should I ship them both, or just use dashboard one?